subsys/debug/Kconfig

```
@@ -421,3 +421,12 @@ config GDBSTUB_BUF_SZ
 	  full GDB packet at a time.
 
 endif
+
+config USE_SEGGER_DEBUGMON
```

`USE_` is redundant.

```suggestion
config SEGGER_DEBUGMON
```